### PR TITLE
[Merged by Bors] - Fix local testnet scripts

### LINKED ISSUE #2094: Local tesnet with one beacon node
## Description

After following instruction from [scripts/local_testnet](https://github.com/sigp/lighthouse/tree/unstable/scripts/local_testnet) block is not initialized with one beacon node

## Version

branch `unstable` (Lighthouse/v1.0.4-c1e27f4+)

## Present Behaviour

running one beacon node and validator in validator got an error `SERVICE_UNAVAILABLE: beacon node is syncing: sync is stalled`

<details>
  <summary>Beacon Node log</summary>

> Dec 15 12:20:28.253 INFO Lighthouse started                      version: Lighthouse/v1.0.4-c1e27f4+
Dec 15 12:20:28.253 INFO Configured for network                  name: custom (/home/bernard/rust/lighthouse/scripts/local_testnet/testnet)
Dec 15 12:20:28.253 INFO Data directory initialised              datadir: /home/bernard/rust/lighthouse/scripts/local_testnet/beacon
Dec 15 12:20:28.259 INFO Deposit contract                        address: 0x1234567890123456789012345678901234567890, deploy_block: 0
Dec 15 12:20:28.273 INFO Starting from known genesis state       service: beacon
Dec 15 12:20:28.363 WARN Block production impaired               reason: dummy eth1 backend is enabled
Dec 15 12:20:28.371 INFO Beacon chain initialized                head_slot: 0, head_block: 0x94f5…140e, head_state: 0x4334…b1fd, service: beacon
Dec 15 12:20:28.371 INFO Timer service started                   service: node_timer
Dec 15 12:20:28.371 INFO UPnP Attempting to initialise routes    service: UPnP
Dec 15 12:20:28.372 INFO Libp2p Service                          peer_id: 16Uiu2HAmLJ99ydPPmaRNLhCtuBuUu4hd7MgtGSr9QpeB2WWn7s5b, service: libp2p
Dec 15 12:20:28.372 INFO ENR Initialised                         tcp: Some(9000), udp: Some(9000), ip: Some(127.0.0.1), id: 0x6d8a..ba37, seq: 1, enr: enr:-LK4QCNQ3Kdn_eTptzzK78ybzcmjKYtVI5ElDlJz5nzPVsGxYmWfMisCAnjq44voBDzoegNB7vJgqv86BaGJORH9S6IBh2F0dG5ldHOIAAAAAAAAAACEZXRoMpC7FlCLAAAAAP__________gmlkgnY0gmlwhH8AAAGJc2VjcDI1NmsxoQNxhkEUD06RzujimP8xzRHVFYmMivfm1AU_bbjx5m1tsoN0Y3CCIyiDdWRwgiMo, service: libp2p
Dec 15 12:20:28.372 INFO Listening established                   address: /ip4/0.0.0.0/tcp/9000/p2p/16Uiu2HAmLJ99ydPPmaRNLhCtuBuUu4hd7MgtGSr9QpeB2WWn7s5b, service: libp2p
Dec 15 12:20:28.378 INFO HTTP API started                        listen_address: 127.0.0.1:5052
Dec 15 12:20:37.001 WARN Low peer count                          peer_count: 0, service: slot_notifier
Dec 15 12:20:37.001 INFO Searching for peers                     current_slot: 1, head_slot: 0, finalized_epoch: 0, finalized_root: 0x0000…0000, peers: 0, service: slot_notifier
Dec 15 12:20:37.001 WARN Syncing eth1 block cache                est_blocks_remaining: initializing deposits, msg: sync can take longer when using remote eth1 nodes, service: slot_notifier
Dec 15 12:20:40.336 WARN Error processing HTTP API request       method: GET, path: /eth/v1/validator/duties/proposer/0, status: 503 Service Unavailable, elapsed: 47.301µs
Dec 15 12:20:40.338 WARN Error processing HTTP API request       method: POST, path: /eth/v1/validator/duties/attester/1, status: 503 Service Unavailable, elapsed: 27.934µs
Dec 15 12:20:43.120 WARN Error processing HTTP API request       method: GET, path: /eth/v1/validator/duties/proposer/0, status: 503 Service Unavailable, elapsed: 238.904µs
Dec 15 12:20:43.135 WARN Error processing HTTP API request       method: POST, path: /eth/v1/validator/duties/attester/1, status: 503 Service Unavailable, elapsed: 111.681µs
Dec 15 12:20:49.001 WARN Low peer count                          peer_count: 0, service: slot_notifier
Dec 15 12:20:49.001 INFO Searching for peers                     current_slot: 2, head_slot: 0, finalized_epoch: 0, finalized_root: 0x0000…0000, peers: 0, service: slot_notifier
Dec 15 12:20:49.001 WARN Syncing eth1 block cache                est_blocks_remaining: initializing deposits, msg: sync can take longer when using remote eth1 nodes, service: slot_notifier
Dec 15 12:20:55.117 WARN Error processing HTTP API request       method: GET, path: /eth/v1/validator/duties/proposer/0, status: 503 Service Unavailable, elapsed: 162.314µs
Dec 15 12:20:55.124 WARN Error processing HTTP API request       method: POST, path: /eth/v1/validator/duties/attester/1, status: 503 Service Unavailable, elapsed: 54.065µs
Dec 15 12:21:01.001 WARN Low peer count                          peer_count: 0, service: slot_notifier
Dec 15 12:21:01.002 INFO Searching for peers                     current_slot: 3, head_slot: 0, finalized_epoch: 0, finalized_root: 0x0000…0000, peers: 0, service: slot_notifier
Dec 15 12:21:01.002 WARN Syncing eth1 block cache                est_blocks_remaining: initializing deposits, msg: sync can take longer when using remote eth1 nodes, service: slot_notifier
Dec 15 12:21:07.117 WARN Error processing HTTP API request       method: GET, path: /eth/v1/validator/duties/proposer/0, status: 503 Service Unavailable, elapsed: 298.466µs
Dec 15 12:21:07.130 WARN Error processing HTTP API request       method: POST, path: /eth/v1/validator/duties/attester/1, status: 503 Service Unavailable, elapsed: 80.911µs
Dec 15 12:21:13.001 WARN Low peer count                          peer_count: 0, service: slot_notifier
Dec 15 12:21:13.001 INFO Searching for peers                     current_slot: 4, head_slot: 0, finalized_epoch: 0, finalized_root: 0x0000…0000, peers: 0, service: slot_notifier
Dec 15 12:21:13.002 WARN Syncing eth1 block cache                est_blocks_remaining: initializing deposits, msg: sync can take longer when using remote eth1 nodes, service: slot_notifier
Dec 15 12:21:19.116 WARN Error processing HTTP API request       method: GET, path: /eth/v1/validator/duties/proposer/0, status: 503 Service Unavailable, elapsed: 160.057µs
Dec 15 12:21:19.123 WARN Error processing HTTP API request       method: POST, path: /eth/v1/validator/duties/attester/1, status: 503 Service Unavailable, elapsed: 57.953µs
Dec 15 12:21:25.001 WARN Low peer count                          peer_count: 0, service: slot_notifier
Dec 15 12:21:25.001 INFO Searching for peers                     current_slot: 5, head_slot: 0, finalized_epoch: 0, finalized_root: 0x0000…0000, peers: 0, service: slot_notifier
Dec 15 12:21:25.002 WARN Syncing eth1 block cache                est_blocks_remaining: initializing deposits, msg: sync can take longer when using remote eth1 nodes, service: slot_notifier
Dec 15 12:21:31.118 WARN Error processing HTTP API request       method: GET, path: /eth/v1/validator/duties/proposer/0, status: 503 Service Unavailable, elapsed: 231.477µs
Dec 15 12:21:31.131 WARN Error processing HTTP API request       method: POST, path: /eth/v1/validator/duties/attester/1, status: 503 Service Unavailable, elapsed: 122.051µs
Dec 15 12:21:37.000 WARN Low peer count                          peer_count: 0, service: slot_notifier
Dec 15 12:21:37.000 INFO Searching for peers                     current_slot: 6, head_slot: 0, finalized_epoch: 0, finalized_root: 0x0000…0000, peers: 0, service: slot_notifier
Dec 15 12:21:37.001 WARN Syncing eth1 block cache                est_blocks_remaining: initializing deposits, msg: sync can take longer when using remote eth1 nodes, service: slot_notifier
Dec 15 12:21:43.104 WARN Error processing HTTP API request       method: GET, path: /eth/v1/validator/duties/proposer/0, status: 503 Service Unavailable, elapsed: 46.547µs
Dec 15 12:21:43.106 WARN Error processing HTTP API request       method: POST, path: /eth/v1/validator/duties/attester/1, status: 503 Service Unavailable, elapsed: 45.833µs
Dec 15 12:21:49.001 WARN Low peer count                          peer_count: 0, service: slot_notifier
Dec 15 12:21:49.001 INFO Searching for peers                     current_slot: 7, head_slot: 0, finalized_epoch: 0, finalized_root: 0x0000…0000, peers: 0, service: slot_notifier
Dec 15 12:21:49.001 WARN Syncing eth1 block cache                est_blocks_remaining: initializing deposits, msg: sync can take longer when using remote eth1 nodes, service: slot_notifier
Dec 15 12:21:55.118 WARN Error processing HTTP API request       method: GET, path: /eth/v1/validator/duties/proposer/0, status: 503 Service Unavailable, elapsed: 270.074µs
Dec 15 12:21:55.126 WARN Error processing HTTP API request       method: POST, path: /eth/v1/validator/duties/attester/1, status: 503 Service Unavailable, elapsed: 53.043µs
Dec 15 12:22:01.001 WARN Low peer count                          peer_count: 0, service: slot_notifier
Dec 15 12:22:01.001 INFO Searching for peers                     current_slot: 8, head_slot: 0, finalized_epoch: 0, finalized_root: 0x0000…0000, peers: 0, service: slot_notifier
Dec 15 12:22:01.002 WARN Syncing eth1 block cache                est_blocks_remaining: initializing deposits, msg: sync can take longer when using remote eth1 nodes, service: slot_notifier
Dec 15 12:22:07.121 WARN Error processing HTTP API request       method: GET, path: /eth/v1/validator/duties/proposer/0, status: 503 Service Unavailable, elapsed: 246.72µs
Dec 15 12:22:07.136 WARN Error processing HTTP API request       method: POST, path: /eth/v1/validator/duties/attester/1, status: 503 Service Unavailable, elapsed: 110.739µs
Dec 15 12:22:13.001 WARN Low peer count                          peer_count: 0, service: slot_notifier
Dec 15 12:22:13.002 INFO Searching for peers                     current_slot: 9, head_slot: 0, finalized_epoch: 0, finalized_root: 0x0000…0000, peers: 0, service: slot_notifier
Dec 15 12:22:13.002 WARN Syncing eth1 block cache                est_blocks_remaining: initializing deposits, msg: sync can take longer when using remote eth1 nodes, service: slot_notifier
Dec 15 12:22:19.121 WARN Error processing HTTP API request       method: GET, path: /eth/v1/validator/duties/proposer/0, status: 503 Service Unavailable, elapsed: 240.744µs
Dec 15 12:22:19.135 WARN Error processing HTTP API request       method: POST, path: /eth/v1/validator/duties/attester/1, status: 503 Service Unavailable, elapsed: 107.167µs
Dec 15 12:22:25.001 WARN Low peer count                          peer_count: 0, service: slot_notifier
Dec 15 12:22:25.002 INFO Searching for peers                     current_slot: 10, head_slot: 0, finalized_epoch: 0, finalized_root: 0x0000…0000, peers: 0, service: slot_notifier
Dec 15 12:22:25.003 WARN Syncing eth1 block cache                est_blocks_remaining: initializing deposits, msg: sync can take longer when using remote eth1 nodes, service: slot_notifier
Dec 15 12:22:31.123 WARN Error processing HTTP API request       method: GET, path: /eth/v1/validator/duties/proposer/0, status: 503 Service Unavailable, elapsed: 478.06µs
Dec 15 12:22:31.145 WARN Error processing HTTP API request       method: POST, path: /eth/v1/validator/duties/attester/1, status: 503 Service Unavailable, elapsed: 124.869µs
Dec 15 12:22:37.001 WARN Low peer count                          peer_count: 0, service: slot_notifier
Dec 15 12:22:37.002 INFO Searching for peers                     current_slot: 11, head_slot: 0, finalized_epoch: 0, finalized_root: 0x0000…0000, peers: 0, service: slot_notifier
Dec 15 12:22:37.002 WARN Syncing eth1 block cache                est_blocks_remaining: initializing deposits, msg: sync can take longer when using remote eth1 nodes, service: slot_notifier
Dec 15 12:22:39.144 WARN Error processing HTTP API request       method: GET, path: /favicon.ico, status: 405 Method Not Allowed, elapsed: 94.218µs
Dec 15 12:22:43.121 WARN Error processing HTTP API request       method: GET, path: /eth/v1/validator/duties/proposer/0, status: 503 Service Unavailable, elapsed: 236.405µs
Dec 15 12:22:43.137 WARN Error processing HTTP API request       method: POST, path: /eth/v1/validator/duties/attester/1, status: 503 Service Unavailable, elapsed: 111.95µs
Dec 15 12:22:49.001 WARN Low peer count                          peer_count: 0, service: slot_notifier
Dec 15 12:22:49.001 INFO Searching for peers                     current_slot: 12, head_slot: 0, finalized_epoch: 0, finalized_root: 0x0000…0000, peers: 0, service: slot_notifier
Dec 15 12:22:49.001 WARN Syncing eth1 block cache                est_blocks_remaining: initializing deposits, msg: sync can take longer when using remote eth1 nodes, service: slot_notifier
Dec 15 12:22:55.121 WARN Error processing HTTP API request       method: GET, path: /eth/v1/validator/duties/proposer/0, status: 503 Service Unavailable, elapsed: 236.528µs
Dec 15 12:22:55.137 WARN Error processing HTTP API request       method: POST, path: /eth/v1/validator/duties/attester/1, status: 503 Service Unavailable, elapsed: 114.329µs
^CDec 15 12:22:59.817 INFO Shutting down..
Dec 15 12:22:59.817 INFO Saved DHT state                         service: network
Dec 15 12:22:59.817 INFO Network service shutdown                service: network
Dec 15 12:22:59.818 INFO Saved beacon chain to disk              service: beacon

</details>

<details>
  <summary>Validator log</summary>

> Dec 15 12:20:39.372 INFO Lighthouse started                      version: Lighthouse/v1.0.4-c1e27f4+
Dec 15 12:20:39.373 INFO Configured for network                  name: custom (/home/bernard/rust/lighthouse/scripts/local_testnet/testnet)
Dec 15 12:20:39.373 INFO Starting validator client               validator_dir: "/home/bernard/rust/lighthouse/scripts/local_testnet/validators", beacon_node: http://localhost:5052/
Dec 15 12:20:39.373 INFO HTTP metrics server is disabled
Dec 15 12:20:39.386 INFO Completed validator discovery           new_validators: 15
WARN: Scrypt parameters are too weak (n: 2, p: 1, r: 8), we recommend (n: 262144, p: 1, r: 8)
WARN: Scrypt parameters are too weak (n: 2, p: 1, r: 8), we recommend (n: 262144, p: 1, r: 8)
Dec 15 12:20:39.393 INFO Enabled validator                       voting_pubkey: 0x84398f539a64cbe01cfcd8c485ea51cd6657b94df93ee9b5dc61e1f18f69da6ca9d4dba63c956a81c68d5d4d4277a60f
Dec 15 12:20:39.393 INFO Enabled validator                       voting_pubkey: 0xab0bdda0f85f842f431beaccf1250bf1fd7ba51b4100fd64364b6401fda85bb0069b3e715b58819684e7fc0b10a72a34
WARN: Scrypt parameters are too weak (n: 2, p: 1, r: 8), we recommend (n: 262144, p: 1, r: 8)
Dec 15 12:20:39.394 INFO Enabled validator                       voting_pubkey: 0x9977f1c8b731a8d5558146bfb86caea26434f3c5878b589bf280a42c9159e700e9df0e4086296c20b011d2e78c27d373
WARN: Scrypt parameters are too weak (n: 2, p: 1, r: 8), we recommend (n: 262144, p: 1, r: 8)
Dec 15 12:20:39.395 INFO WARN: Scrypt parameters are too weak (n: 2, p: 1, r: 8), we recommend (n: 262144, p: 1, r: 8)
Enabled validator                       voting_pubkey: 0x876dd4705157eb66dc71bc2e07fb151ea53e1a62a0bb980a7ce72d15f58944a8a3752d754f52f4a60dbfc7b18169f268
Dec 15 12:20:39.395 INFO Enabled validator                       voting_pubkey: 0x903e2989e7442ee0a8958d020507a8bd985d3974f5e8273093be00db3935f0500e141b252bd09e3728892c7a8443863c
WARN: Scrypt parameters are too weak (n: 2, p: 1, r: 8), we recommend (n: 262144, p: 1, r: 8)
Dec 15 12:20:39.396 INFO Enabled validator                       voting_pubkey: 0xa8d4c7c27795a725961317ef5953a7032ed6d83739db8b0e8a72353d1b8b4439427f7efa2c89caa03cc9f28f8cbab8ac
WARN: Scrypt parameters are too weak (n: 2, p: 1, r: 8), we recommend (n: 262144, p: 1, r: 8)
Dec 15 12:20:39.396 INFO Enabled validator                       voting_pubkey: 0x88c141df77cd9d8d7a71a75c826c41a9c9f03c6ee1b180f3e7852f6a280099ded351b58d66e653af8e42816a4d8f532e
WARN: Scrypt parameters are too weak (n: 2, p: 1, r: 8), we recommend (n: 262144, p: 1, r: 8)
WARN: Scrypt parameters are too weak (n: 2, p: 1, r: 8), we recommend (n: 262144, p: 1, r: 8)
Dec 15 12:20:39.396 INFO Enabled validator                       voting_pubkey: 0xb89bebc699769726a318c8e9971bd3171297c61aea4a6578a7a4f94b547dcba5bac16a89108b6b6a1fe3695d1a874a0b
Dec 15 12:20:39.397 INFO Enabled validator                       voting_pubkey:WARN: Scrypt parameters are too weak (n: 2, p: 1, r: 8), we recommend (n: 262144, p: 1, r: 8)
 0x81283b7a20e1ca460ebd9bbd77005d557370cabb1f9a44f530c4c4c66230f675f8df8b4c2818851aa7d77a80ca5a4a5e
Dec 15 12:20:39.397 INFO Enabled validatorWARN: Scrypt parameters are too weak (n: 2, p: 1, r: 8), we recommend (n: 262144, p: 1, r: 8)
                       voting_pubkey: 0x9893413c00283a3f9ed9fd9845dda1cea38228d22567f9541dccc357e54a2d6a6e204103c92564cbc05f4905ac7c493a
Dec 15 12:20:39.398 INFO Enabled validatorWARN: Scrypt parameters are too weak (n: 2, p: 1, r: 8), we recommend (n: 262144, p: 1, r: 8)
                       voting_pubkey: 0xa99a76ed7796f7be22d5b7e85deeb7c5677e88e511e0b337618f8c4eb61349b4bf2d153f649f7b53359fe8b94a38e44c
Dec 15 12:20:39.398 INFO Enabled validatorWARN: Scrypt parameters are too weak (n: 2, p: 1, r: 8), we recommend (n: 262144, p: 1, r: 8)
                       voting_pubkey: 0x9314c6de0386635e2799af798884c2ea09c63b9f079e572acc00b06a7faccce501ea4dfc0b1a23b8603680a5e3481327
Dec 15 12:20:39.398 INFO WARN: Scrypt parameters are too weak (n: 2, p: 1, r: 8), we recommend (n: 262144, p: 1, r: 8)
Enabled validator                       voting_pubkey: 0xaec922bd7a9b7b1dc21993133b586b0c3041c1e2e04b513e862227b9d7aecaf9444222f7e78282a449622ffc6278915d
Dec 15 12:20:39.399 INFO Enabled validator                       voting_pubkeyWARN: Scrypt parameters are too weak (n: 2, p: 1, r: 8), we recommend (n: 262144, p: 1, r: 8)
: 0xa6d310dbbfab9a22450f59993f87a4ce5db6223f3b5f1f30d2c4ec718922d400e0b3c7741de8e59960f72411a0ee10a7
Dec 15 12:20:39.399 INFO Enabled validator                       voting_pubkey: 0xa3a32b0f8b4ddb83f1a0a853d81dd725dfe577d4f4c3db8ece52ce2b026eca84815c1a7e8e92a4de3d755733bf7e4a9b
Dec 15 12:20:40.318 INFO Modified key_cache saved successfully
Dec 15 12:20:40.318 INFO Initialized validators                  enabled: 15, disabled: 0
Dec 15 12:20:40.334 INFO Connected to beacon node                version: Lighthouse/v1.0.4-c1e27f4+/x86_64-linux
Dec 15 12:20:40.334 INFO Loaded validator keypair store          voting_validators: 15
Dec 15 12:20:40.334 INFO Genesis has already occurred            seconds_ago: 21
Dec 15 12:20:40.334 INFO Block production service started        service: block
Dec 15 12:20:40.334 INFO Attestation production service started  next_update_millis: 2665, service: attestation
Dec 15 12:20:40.334 INFO HTTP API server is disabled
Dec 15 12:20:40.336 ERRO Failed to download validator duties     error: Failed to get proposer indices: ServerMessage(ErrorMessage { code: 503, message: "SERVICE_UNAVAILABLE: beacon node is syncing: sync is stalled", stacktraces: [] }), service: duties
Dec 15 12:20:40.338 ERRO Failed to download validator duties     error: Failed to get attester duties: ServerMessage(ErrorMessage { code: 503, message: "SERVICE_UNAVAILABLE: beacon node is syncing: sync is stalled", stacktraces: [] }), service: duties
Dec 15 12:20:43.120 ERRO Failed to download validator duties     error: Failed to get proposer indices: ServerMessage(ErrorMessage { code: 503, message: "SERVICE_UNAVAILABLE: beacon node is syncing: sync is stalled", stacktraces: [] }), service: duties
Dec 15 12:20:43.136 ERRO Failed to download validator duties     error: Failed to get attester duties: ServerMessage(ErrorMessage { code: 503, message: "SERVICE_UNAVAILABLE: beacon node is syncing: sync is stalled", stacktraces: [] }), service: duties
Dec 15 12:20:49.003 INFO Awaiting activation                     slot: 2, epoch: 0, validators: 15, service: notifier
Dec 15 12:20:55.117 ERRO Failed to download validator duties     error: Failed to get proposer indices: ServerMessage(ErrorMessage { code: 503, message: "SERVICE_UNAVAILABLE: beacon node is syncing: sync is stalled", stacktraces: [] }), service: duties
Dec 15 12:20:55.124 ERRO Failed to download validator duties     error: Failed to get attester duties: ServerMessage(ErrorMessage { code: 503, message: "SERVICE_UNAVAILABLE: beacon node is syncing: sync is stalled", stacktraces: [] }), service: duties
Dec 15 12:21:01.003 INFO Awaiting activation                     slot: 3, epoch: 0, validators: 15, service: notifier
Dec 15 12:21:07.118 ERRO Failed to download validator duties     error: Failed to get proposer indices: ServerMessage(ErrorMessage { code: 503, message: "SERVICE_UNAVAILABLE: beacon node is syncing: sync is stalled", stacktraces: [] }), service: duties
Dec 15 12:21:07.130 ERRO Failed to download validator duties     error: Failed to get attester duties: ServerMessage(ErrorMessage { code: 503, message: "SERVICE_UNAVAILABLE: beacon node is syncing: sync is stalled", stacktraces: [] }), service: duties
Dec 15 12:21:13.002 INFO Awaiting activation                     slot: 4, epoch: 0, validators: 15, service: notifier
Dec 15 12:21:19.117 ERRO Failed to download validator duties     error: Failed to get proposer indices: ServerMessage(ErrorMessage { code: 503, message: "SERVICE_UNAVAILABLE: beacon node is syncing: sync is stalled", stacktraces: [] }), service: duties
Dec 15 12:21:19.124 ERRO Failed to download validator duties     error: Failed to get attester duties: ServerMessage(ErrorMessage { code: 503, message: "SERVICE_UNAVAILABLE: beacon node is syncing: sync is stalled", stacktraces: [] }), service: duties
Dec 15 12:21:25.002 INFO Awaiting activation                     slot: 5, epoch: 0, validators: 15, service: notifier
Dec 15 12:21:31.119 ERRO Failed to download validator duties     error: Failed to get proposer indices: ServerMessage(ErrorMessage { code: 503, message: "SERVICE_UNAVAILABLE: beacon node is syncing: sync is stalled", stacktraces: [] }), service: duties
Dec 15 12:21:31.132 ERRO Failed to download validator duties     error: Failed to get attester duties: ServerMessage(ErrorMessage { code: 503, message: "SERVICE_UNAVAILABLE: beacon node is syncing: sync is stalled", stacktraces: [] }), service: duties
Dec 15 12:21:37.001 INFO Awaiting activation                     slot: 6, epoch: 0, validators: 15, service: notifier
Dec 15 12:21:43.104 ERRO Failed to download validator duties     error: Failed to get proposer indices: ServerMessage(ErrorMessage { code: 503, message: "SERVICE_UNAVAILABLE: beacon node is syncing: sync is stalled", stacktraces: [] }), service: duties
Dec 15 12:21:43.106 ERRO Failed to download validator duties     error: Failed to get attester duties: ServerMessage(ErrorMessage { code: 503, message: "SERVICE_UNAVAILABLE: beacon node is syncing: sync is stalled", stacktraces: [] }), service: duties
Dec 15 12:21:49.002 INFO Awaiting activation                     slot: 7, epoch: 0, validators: 15, service: notifier
Dec 15 12:21:55.118 ERRO Failed to download validator duties     error: Failed to get proposer indices: ServerMessage(ErrorMessage { code: 503, message: "SERVICE_UNAVAILABLE: beacon node is syncing: sync is stalled", stacktraces: [] }), service: duties
Dec 15 12:21:55.126 ERRO Failed to download validator duties     error: Failed to get attester duties: ServerMessage(ErrorMessage { code: 503, message: "SERVICE_UNAVAILABLE: beacon node is syncing: sync is stalled", stacktraces: [] }), service: duties
Dec 15 12:22:01.002 INFO Awaiting activation                     slot: 8, epoch: 0, validators: 15, service: notifier
Dec 15 12:22:07.122 ERRO Failed to download validator duties     error: Failed to get proposer indices: ServerMessage(ErrorMessage { code: 503, message: "SERVICE_UNAVAILABLE: beacon node is syncing: sync is stalled", stacktraces: [] }), service: duties
Dec 15 12:22:07.136 ERRO Failed to download validator duties     error: Failed to get attester duties: ServerMessage(ErrorMessage { code: 503, message: "SERVICE_UNAVAILABLE: beacon node is syncing: sync is stalled", stacktraces: [] }), service: duties
Dec 15 12:22:13.003 INFO Awaiting activation                     slot: 9, epoch: 0, validators: 15, service: notifier
Dec 15 12:22:19.122 ERRO Failed to download validator duties     error: Failed to get proposer indices: ServerMessage(ErrorMessage { code: 503, message: "SERVICE_UNAVAILABLE: beacon node is syncing: sync is stalled", stacktraces: [] }), service: duties
Dec 15 12:22:19.135 ERRO Failed to download validator duties     error: Failed to get attester duties: ServerMessage(ErrorMessage { code: 503, message: "SERVICE_UNAVAILABLE: beacon node is syncing: sync is stalled", stacktraces: [] }), service: duties
Dec 15 12:22:25.003 INFO Awaiting activation                     slot: 10, epoch: 0, validators: 15, service: notifier
Dec 15 12:22:31.124 ERRO Failed to download validator duties     error: Failed to get proposer indices: ServerMessage(ErrorMessage { code: 503, message: "SERVICE_UNAVAILABLE: beacon node is syncing: sync is stalled", stacktraces: [] }), service: duties
Dec 15 12:22:31.145 ERRO Failed to download validator duties     error: Failed to get attester duties: ServerMessage(ErrorMessage { code: 503, message: "SERVICE_UNAVAILABLE: beacon node is syncing: sync is stalled", stacktraces: [] }), service: duties
Dec 15 12:22:37.003 INFO Awaiting activation                     slot: 11, epoch: 0, validators: 15, service: notifier
Dec 15 12:22:43.121 ERRO Failed to download validator duties     error: Failed to get proposer indices: ServerMessage(ErrorMessage { code: 503, message: "SERVICE_UNAVAILABLE: beacon node is syncing: sync is stalled", stacktraces: [] }), service: duties
Dec 15 12:22:43.138 ERRO Failed to download validator duties     error: Failed to get attester duties: ServerMessage(ErrorMessage { code: 503, message: "SERVICE_UNAVAILABLE: beacon node is syncing: sync is stalled", stacktraces: [] }), service: duties
Dec 15 12:22:49.003 INFO Awaiting activation                     slot: 12, epoch: 0, validators: 15, service: notifier
Dec 15 12:22:55.121 ERRO Failed to download validator duties     error: Failed to get proposer indices: ServerMessage(ErrorMessage { code: 503, message: "SERVICE_UNAVAILABLE: beacon node is syncing: sync is stalled", stacktraces: [] }), service: duties
Dec 15 12:22:55.137 ERRO Failed to download validator duties     error: Failed to get attester duties: ServerMessage(ErrorMessage { code: 503, message: "SERVICE_UNAVAILABLE: beacon node is syncing: sync is stalled", stacktraces: [] }), service: duties
^CDec 15 12:22:57.971 INFO Shutting down..

</details>

## Expected Behaviour

able to run testnet beacon node and validator without getting stalled sync (like when 2 beacon nodes are started)

### PR DESCRIPTION
**Pull request by pawanjay176**
## Issue Addressed

Resolves #2094 

## Proposed Changes

Fixes scripts for creating local testnets. Adds an option in `lighthouse boot_node` to run with a previously generated enr.

### CODE REVIEW

File: scripts/local_testnet/ganache_test_node.sh [hunk 1]
```diff
@@ -1,8 +1,13 @@
 #!/usr/bin/env bash
 
+source ./vars.env
+
 ganache-cli \
 	--defaultBalanceEther 1000000000 \
 	--gasLimit 1000000000 \
 	--accounts 10 \
-	--mnemonic "vast thought differ pull jewel broom cook wrist tribe word before omit" \
+	--mnemonic $ETH1_NETWORK_MNEMONIC \
```
Comment (ghost) [inline scripts/local_testnet/ganache_test_node.sh]: We may want to enclose this bash variable in quotes, as in: `--mnemonic "$ETH1_NETWORK_MNEMONIC" \`, to get all the words of the mnemonic as opposed to only the first one.

Comment (pawanjay176) [reply scripts/local_testnet/ganache_test_node.sh]: Thanks a lot, was wondering what went wrong with eth1.

File: boot_node/src/cli.rs [hunk 2]
```diff
@@ -57,4 +60,11 @@ pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
                 .help("Discovery can automatically update the node's local ENR with an external IP address and port as seen by other peers on the network. \
                 This enables this feature.")
         )
+        .arg(
+            Arg::with_name("network-dir")
+            .value_name("NETWORK_DIR")
+                .long("network-dir")
+                .help("The directory which contains the enr and it's assoicated private key")
```
Comment (ghost) [inline boot_node/src/cli.rs:67]: Maybe `The directory which contains the enr and its associated private key`

**Review by paulhauner — APPROVED** [review]
Great stuff, this looks like it was a lot of work!

bors r+

**Pull request closed without merging**